MonoGame.Framework/Graphics/States/BlendState.OpenGL.cs

```diff
@@ -14,43 +14,78 @@ namespace Microsoft.Xna.Framework.Graphics
 {
     public partial class BlendState
     {
-        internal void PlatformApplyState(GraphicsDevice device)
+        private static bool _currentBlendEnable = false;
+        private static BlendState _currentBlendState = new BlendState();
```

Comment:
@tgjones - So considering your work in https://github.com/mono/MonoGame/pull/3472, how do you think something like this should be done?  I suspect state like this just needs to live in `GraphicsDevice`.